cmd/inspector/main.go

```diff
@@ -553,22 +553,25 @@ func getSegments(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return ErrInspectorDial.Wrap(err)
 	}
-
-	length := irreparableLimit
-	var offset int32
+	var lastSeenSegmentPath = []byte{}
```

Comment:
why isn't this a `storj.Path`, `string` or something similar?

Reply:
yeah i guess i could have. I didn't even consider that though since it would just add a bunch of code to convert between string and []byte. Maybe that is worth it? im no sure.